src/Configuration/Option/Prefix.php

```diff
@@ -28,7 +29,7 @@ class Prefix implements OptionInterface
      */
     public function filter(OptionsInterface $options, $value)
     {
-        if (is_callable($value)) {
+        if (is_callable($value) && ! function_exists($value)) {
```

Comment:
@mmodler Great, let's add a new test case that `"chr"` won't be called as a function and we can merge it.